File: Two/basic.php
<?php

require_once 'Plan.php';

class Basic extends Plan {
    public function __construct(float $price) {
        parent::__construct("Basic", $price);
    }

    public function getServices(): array {
        return ["Access to all exercise machines"];
    }

    public function calculateCost(): float {
        return $this->getPrice();
    }
}

?>

File: java/Two/Two/Basic.java
public class Basic extends Plan {
    public Basic(float price) {
        super("Basic", price);
    }

    @Override
    public String[] getServices() {
        return new String[]{"Access to all exercise machines"};
    }

    @Override
    public float calculateCost() {
        return getPrice();
    }
}